OneSignalSDK/onesignal/notifications/build.gradle

```diff
@@ -74,8 +74,8 @@ dependencies {
     // firebase-messaging:23.0.0 incoporates fix for SecurityException: Not allowed to bind to service
     api('com.google.firebase:firebase-messaging') {
         version {
-            require '[19.0.0, 23.0.99]'
-            prefer '23.0.8'
+            require '[19.0.0, 23.99.99]'
```

Comment:
I would recommend bumping only the minor version number in this version range. We have seen Firebase and Google break things in minor version bumps in the past.  So thinking this should be `[19.0.0, 23.4.99]`.

Reply:
That makes sense, I pushed the change just now.